doc/en/user/source/data/raster/imagemosaic/configuration.rst

```diff
@@ -206,35 +206,35 @@ In addition to the required envelope and location attributes, the schema for the
      - N
      - Path to an auxiliary file to be used for internal purposes (For example: when dealing with NetCDF granules, it refers to the NetCDF XML ancillary file.)
    * - AbsolutePath
-     - Y
-     - Controls whether or not the path stored inside the ``location`` attribute represents an absolute path or a path relative to the location of the shapefile index. Notice that a relative index ensures much more portability of the mosaic itself. Default value for this parameter is ``false``, which means relative paths.
+     - N
+     - Controls whether or not the path stored inside the ``location`` attribute represents an absolute path or a path relative to the location of the shapefile index. Notice that a relative index ensures better portability of the mosaic itself. Default value for this parameter is ``false``, which means relative paths.
    * - Caching
      - N
-     - Boolean value to enable caching. When set to ``true``, the imagemosaic will try to save in memory the entire contents of the index to reduce loading/query time. Set to ``false`` for a large granule index and/or if new granules are to be ingested (for example, when the index is on a database and we interact directly with it). Default is ``false``.
+     - Boolean value to enable caching. When set to ``true``, the ImageMosaic will try to save in memory the entire contents of the index to reduce loading/query time. Set to ``false`` for a large granule index and/or if new granules are to be ingested (for example, when the index is on a database and we interact directly with it). Default is ``false``.
    * - CanBeEmpty
      - N
-     - Boolean flag used for configuring empty mosaics. When enabled the imagemosaic will not throw an exception caused by the absence of any coverage. By default it is set to false.
+     - Boolean flag used for configuring empty mosaics. When enabled the ImageMosaic will not throw an exception caused by the absence of any coverage. By default it is set to ``false``.
```

Comment:
In a number of places, I have added double-backticks around boolean values for consistency. Mostly, the original document had them in backticks, so I tried treat all instances the same.